src/Examples/UnitModels/Operations/feed_flash_testing.ipynb

```diff
@@ -0,0 +1,468 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tutorial: Feed Flash Unit Model with Modular Property Package\n",
+    "\n",
+    "![](feed.svg)\n",
+    "\n",
+    "## Learning Outcomes\n",
+    "\n",
+    "- Demonstrate use of the Feed Flash unit model in IDAES\n",
+    "- Demonstrate different options available\n",
+    "\n",
+    "\n",
+    "## Problem Statement\n",
+    "\n",
+    "In this tutorial, we will be feeding vapor-liquid streams into the feed flash unit model using two different sets of state variables. Like the traditional feed block, the feed flash unit model has no inlets and only one outlet, so the stream information is fed to the model through a feed port rather than an inlet. Unlike the feed block, this unit model is capable of handling phase equilibrium. The feed conditions are as follows:\n",
```

Comment:
A little more explanation of when you you use this over the standard Feed port is required to help users understand the difference. Specifically, which sets of state variables does this make sense for, versus which ones do not need this (I think the difference is whether flowrate and composition are specified for the overall mixture (e.g. normal Feed) versus phase or component flows (Feed Flash), but you should confirm that as a it is a long time since I wrote this).